policies/allowed_vm_sku/policy.json

```diff
@@ -1,14 +1,14 @@
 {
   "properties": {
-    "displayName": "HMCTS Allowed VM SKU sizes",
+    "displayName": "This policy enables you to specify a set of virtual machine size SKUs that can be deployed in HMCTS",
     "policyType": "Custom",
     "mode": "Indexed",
     "parameters": {
-      "allowedSkus": {
+      "allowedSKUs": {
```

Comment:
`listOfAllowedSKUs` maybe, i am not particular though 👌🏽

Reply:
I changed the parameter to listOfAllowedSkus , however it errored with the message "Error: The existing policy parameter(s) 'allowedSkus' were not found in the policy being updated". Reverted it back to the current value and it worked.